includes/class-migration.php

```diff
@@ -897,4 +901,14 @@ public static function migrate_actor_mode() {
 			\update_option( 'activitypub_actor_mode', ACTIVITYPUB_ACTOR_MODE );
 		}
 	}
+
+	/**
+	 * Migrate the advanced settings.
+	 */
+	public static function migrate_advanced_settings() {
+		// Check if one of the advanced settings is set and show the tab.
+		if ( '1' === \get_option( 'activitypub_authorized_fetch' ) || '180' !== \get_option( 'activitypub_outbox_purge_days' ) ) {
+			\update_option( 'activitypub_show_advanced_tab', '1' );
```

Comment:
Could we make this a user meta, too, like for the welcome screen? Otherwise this will be enabled/disabled for every user on the site.

Reply:
It is limited to the admins, but yes, I can change it.